pkgs/cli/src/main.zig

```diff
@@ -390,3 +390,7 @@ pub fn main() !void {
 test {
     @import("std").testing.refAllDeclsRecursive(@This());
 }
+
+test {
+    _ = @import("test/genesis_to_finalization_test.zig");
```

Comment:
this is an integraton test so it needs to be bundles in simtest (create a common file to import these tests in the test/integration.zig (rename the other integration test file and import it as well)